### ISSUE TITLE
Character movement does not feel intuitive

### ISSUE THREAD

**Issue by Tybobobo** [Hirnbix/captain-holetooth#44]
The character and shop movement should be improved in such a way that the player wont feel restricted to "realism". 
Example of good character movement would be Mario from practically any nintendo game.

How to improve:
- Character only moves (ground/air) when you are pressing movement buttons
- Character can change direction during air time
- Quicker response to movement

**Comment by oliver-ruehl** [comment]
I made some changes to the controls - try it our and see what you think.
Many kids tested and love the controls so far!
We need to keep in mind that kids have not played 23482470 hrs like we did already :)

**Comment by HummusSamurai** [comment]
Is there a particular reason why horizontal movement is slower in the air? It makes platforming unnatural.

I would be willing to fix that and possibly refactor the movement code as it's somewhat overcomplex.

**Comment by oliver-ruehl** [comment]
You are very welcome to fix it, just keep in mind that  it is aimed at kids.
So slow movement is not always a bad thing ;-)
Thanks for taking part in this project!

**Comment by HummusSamurai** [comment]
Definitely, I am not commenting on the overall movement speed, but horizontal movement in air should be equal to that on land.
